test/e2e/e2ethanos/services.go

```diff
@@ -406,18 +406,18 @@ func NewCompactor(sharedDir string, name string, bucketConfig client.BucketConfi
 	return compactor, nil
 }
 
-func NewQueryFrontend(name string, downstreamURL string, respCacheConf cache.ResponseCacheConfig) (*e2e.HTTPService, error) {
-	respCacheConfigBytes, err := yaml.Marshal(respCacheConf)
+func NewQueryFrontend(name string, downstreamURL string, cacheConfig queryfrontend.CacheProviderConfig) (*e2e.HTTPService, error) {
+	cacheConfigBytes, err := yaml.Marshal(cacheConfig)
 	if err != nil {
-		return nil, errors.Wrapf(err, "generate response cache config file: %v", respCacheConf)
+		return nil, errors.Wrapf(err, "marshal query frontend config file: %v", cacheConfig)
```

Comment:
Don't need to change here. This is the response cache config.

Reply:
updated